sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssueFilterParameters.java

```diff
@@ -47,7 +47,12 @@
   public static final String FILE_UUIDS = "fileUuids";
   public static final String ON_COMPONENT_ONLY = "onComponentOnly";
   public static final String RULES = "rules";
-  public static final String ACTION_PLANS = "actionPlans";
+
+  /**
+   * @deprecated since 5.5, action plan feature has been removed
+   */
+  @Deprecated
+  public static final String DEPRECATED_ACTION_PLANS = "actionPlans";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)